guides/common/modules/proc_configuring-rhui-alternate-content-sources.adoc

```diff
@@ -0,0 +1,74 @@
+* Import the client entitlement certificates into your {Project}.
+For more information, see {ContentManagementDocURL}Importing_Custom_SSL_Certificates_content-management[Importing Custom SSL Certificates] in _{ContentManagementDocTitle}_.
+* Obtain a list of the subpaths for the required repositories.
+Execute the following command on the RHUA server:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# rhui-manager repo info --repo_id _My_Repo_ID_
+----
+* Note that the alternate content source paths consist of a base URL appended with the subpaths that you provide.
+For example, if your base URL is `https://server.example.com` and your subpaths are `rhel7/` and `rhel8/`, then both `https://server.example.com/rhel7/` and `https://server.example.com/rhel8/` will be searched. 
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Alternate Content Sources*.
+. Click *Add Source* and set the *Source type* as *RHUI*.
+. Generate RHUI certificates using the command provided in the {ProjectWebUI}.
+Ensure that you pass the repo labels of the desired repositories.
+. In the *Name* field, enter a name for the alternate content source.
+. Optional: In the *Description* field, provide a description for the ACS.
+. Select {SmartProxies} to which the alternate content source is to be synced.
+. Optional: Select `Use HTTP proxies` if you want the ACS to use the {SmartProxy}'s HTTP proxy.
+. Enter the Base URL of the Red Hat Update Infrastructure CDS node.
+. Enter a comma-separated list of Subpaths.
+. Provide the *Content Credentials*, if these are needed.
+. If SSL verification is required, enable *Verify SSL* and select the SSL CA certificate.
+. Review details and click *Add*.
+. Navigate to *Content* > *Alternate Content Sources* > click the vertical ellipsis next to the newly created alternate content source > Select *Refresh*.
+
+[id="cli-configuring-rhui-alternate-content-sources_{context}"]
+.CLI Procedure
+. On {ProjectServer}, enter the following command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer alternate-content-source create \
+--name "_My_ACS_Name_" \
+--alternate-content-source-type rhui \
+--base-url "_https://rhui-cds-node/pulp/content_" \
+--subpaths _path/to/repo/1/, path/to/repo/2/_ \
+--ssl-client-cert-id _SSL_Client_Cert_ID_ \
+--ssl-client-key-id _SSL_Client_Key_ID_ \
+--smart-proxy-ids _{SmartProxy}_ID_
```

Comment:
IMHO the project/product should always verify SSL by default. Is there a particular reason you don't have this default?

Reply:
We did have it as a default before.  It got turned into a required argument in the API when we fixed a bug around verify_ssl being passed silently into the Simplified ACS type, which shouldn't accept it as parameter.

Long story short, it's a bug that we're now requiring verify_ssl. However it'll be that way in the next Katello release so I figured we should document it as such for now.

Reply:
@qcjames53 I'm trying to remember, do we have a Redmine to reinstate the ACS verify-ssl True default?

Reply:
That's fair. I think it's a good practice to link the the bug (Redmine, GH issue) in a code comment. That makes it easier to recognize it's a workaround and to see if it can already be dropped.

Reply:
@ianballou The talk we had with Jeremy was inconclusive. I believe we came to the conclusion that we need to leave verify-ssl as a required parameter for now unless the custom/rhui and simplified ACS's are split into separate classes.

Reply:
There's no redmine at the moment.

Reply:
For completeness, here is the redmine: https://projects.theforeman.org/issues/36187